package.json

```diff
@@ -36,17 +42,17 @@
     "LICENSE"
   ],
   "devDependencies": {
-    "@vitejs/plugin-react": "^3.0.1",
-    "@vitejs/plugin-vue": "^4.0.0",
-    "chromedriver": "^108.0.0",
-    "eslint": "^8.26.0",
-    "mocha": "^9.2.0",
+    "@vitejs/plugin-react": "^3.1.0",
+    "@vitejs/plugin-vue": "^4.1.0",
+    "chromedriver": "^111.0.0",
```

Comment:
Do we need this? Shouldn't this be handled already in a project? What about other browsers?

Reply:
It's the dev dependency, `chromedriver` only exists for the plugin's development.